covalent/_file_transfer/strategies/__init__.py

```diff
@@ -20,3 +20,9 @@
 
 from .http_strategy import HTTP
 from .rsync_strategy import Rsync
+
+try:
+    import boto3
+    from .s3_strategy import S3
+except:
+    print("Kindly install covalent as \"pip install covalent[aws]\" in order to use S3 file transfer strategy.")
```

Comment:
this exception will always show up on default installations.  in this case we want 
```
except:
    pass
```

Reply:
Updated.